src/main.rs

```diff
@@ -183,11 +185,13 @@ async fn main() -> Result<(), anyhow::Error> {
                     .wrap(prometheus.clone())
                     .configure(get_service_routes)
                     .configure(get_basic_routes)
+                    .configure(get_other_service_routes)
             } else {
                 App::new().wrap(prometheus.clone()).service(
                     web::scope(&CONFIG.common.base_uri)
                         .configure(get_service_routes)
-                        .configure(get_basic_routes),
+                        .configure(get_basic_routes)
+                        .configure(get_other_service_routes),
                 )
```

Comment:
This part should keep baseic_routes at the end, the routes have priority.  basic one the top maybe cause problem.